cas/index_gc.c

```diff
@@ -361,6 +361,11 @@ static int cgc_fom_tick(struct m0_fom *fom0)
 			fom->cg_ctg_op_initialized = true;
 			result = m0_ctg_drop(ctg_op, fom->cg_ctg,
 					     CGC_LOCK_DEAD_INDEX);
+			/**
+			 * Free the memory allocated for the root node after
+			 * destroying the tree.
+			 */
+			m0_free0(&fom->cg_ctg->cc_tree);
```

Comment:
Should this be ordinary comment instead of doxygen type?

Reply:
Corrected.